tt_metal/tools/profiler/common.py

```diff
@@ -36,6 +35,18 @@
 TRACY_CSVEXPROT_TOOL = "csvexport-release"
 
 
+def generate_logs_folder(outFolder):
+    return outFolder / ".logs"
```

Comment:
should logs folder be hidden? (starting with . )

Reply:
Yeah, unless you are a dev no one should need to look at the data here. `report` is where the interesting files are presented.